subprojects/composite-builds/src/test/groovy/org/gradle/composite/internal/DefaultIncludedBuildRegistryTest.groovy

```diff
@@ -349,14 +349,14 @@ class DefaultIncludedBuildRegistryTest extends Specification {
     private BuildLifecycleController buildController(SettingsInternal settings, GradleInternal gradle) {
         def buildController = Stub(BuildLifecycleController)
         def services = Stub(ServiceRegistry)
-        def buildLocations = Stub(BuildLocations)
+        def buildLocations = Stub(BuildLayout)
 
         _ * buildController.gradle >> gradle
         if (settings != null) {
             _ * gradle.settings >> settings
         }
         _ * gradle.services >> services
-        _ * services.get(BuildLocations) >> buildLocations
+        _ * services.get(BuildLayout) >> buildLocations
         _ * buildLocations.rootDirectory >> tmpDir.file("root-dir")
```

Comment:
```suggestion
        _ * services.get(BuildLayout) >> buildLayout
        _ * buildLayout.rootDirectory >> tmpDir.file("root-dir")
```

Reply:
Sorry, I missed this in haste, coming up in a follow-up PR.

Reply:
See: https://github.com/gradle/gradle/pull/27063